lib/senders/index.js

```diff
@@ -165,55 +195,89 @@ module.exports = function (log, config, error, db, translator, sender) {
             })
           })
       },
-      sendPasswordChangedNotification: function (email, opts) {
-        return getSafeMailer(email)
-          .then(function (mailer) {
-            return mailer.passwordChangedEmail({
-              email: email,
-              acceptLanguage: opts.acceptLanguage || defaultLanguage,
-              ip: opts.ip,
-              location: opts.location,
-              uaBrowser: opts.uaBrowser,
-              uaBrowserVersion: opts.uaBrowserVersion,
-              uaOS: opts.uaOS,
-              uaOSVersion: opts.uaOSVersion
-            })
-          })
+      sendPasswordChangedNotification: function (emails, opts) {
+        var mailerPromises = emails.map(function (emailRecord) {
```

Comment:
Your instinct is correct. I opted to have it use the `cc` for secondary emails instead of using `to`, which goes to primary. This gives a nice separation for which email does what.